prac_08/convert_miles_km.py

```diff
@@ -0,0 +1,39 @@
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.core.window import Window
+from kivy.properties import StringProperty
+
+MILES_TO_KM_RATE = 1.60934
+
+
+class ConvertMilesKmApp(App):
+    """ ConvertMilesKmApp is a Kivy App for converting miles to Kms."""
+    message = StringProperty()
+
+    def build(self):
+        """ build the Kivy app from the kv file """
+        Window.size = (600, 300)
+        self.title = "Convert miles to Km"
+        self.root = Builder.load_file('convert_miles_km.kv')
+        self.message = ""
+        return self.root
+
+    def handle_increment(self, value, step):
+        """Handle an increment after a button press and output the result back to the text input."""
+        try:
+            new_value = int(value) + step
```

Comment:
If the user typed in a number with a decimal point, such a 5.4km, this would increment the number to 6km. Is this what you want?

Reply:
No, good find thank you.